portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the value entered in the form.
+    String name = request.getParameter("name");
+    String company = request.getParameter("company");
+    String email = request.getParameter("email");
+    String message = request.getParameter("text-input");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("Name: " + name);
```

Comment:
Nit: Totally my preference as well, so feel free to ignore this. Generally I prefer combining print statements whenever possible. If the code is more condensed, it makes it easier to read, and in some instances (probably not in this case so doesn't really matter) having too many print statements causes unnecessary overhead.